scenarios/basics/entity_resolution/SPECIFICATION.md

```diff
@@ -0,0 +1,289 @@
+) is provided to create these resources.
+
+## Hello AWS Entity Resolution
+
+This program is intended for users not familiar with the AWS Entity Resolution
+Service to easily get up and running. The program uses a
+`listMatchingWorkflowsPaginator` to demonstrate how you can read through
+workflow information.
+
+## Basics Scenario Program Flow
+
+The AWS Entity Resolution Basics scenario executes the following operations.
+
+1. **Create a schema mapping**:
+    - Description: Creates a schema mapping by invoking the
+      `createSchemaMapping` method.
+    - Exception Handling: Check to see if a `ConflictException` is thrown, which
+      indicates that the schema mapping already exists. If the exception is
+      thrown, display the information and end the program.
+
+2. **Create a Matching Workflow**:
+    - Description: Creates a new matching workflow that defines how entities
+      should be resolved and matched. The method `createMatchingWorkflow` is
+      called.
+    - Exception Handling: Check to see if a `ConflictException` is thrown, which
+      is thrown if the matching workflow already exists. ALso check to see if a `ValidationException` is thrown. If so, display the message and end the program.
+
+3. **Start Matching Workflow**:
+    - Description: Initiates a matching workflow by calling the
+      `startMatchingJob` method to process entity resolution based on predefined
+      configurations.
+    - Exception Handling: Check to see if an `ConflictException` is thrown,
+      which indicates that the matching workflow job is already running. If the
+      exception is thrown, display the message and end the program.
+
+4. **Get Workflow Job Details**:
+    - Description: Retrieves details about a specific matching workflow job by
+      calling the `getMatchingJob` method.
+    - Exception Handling: Check to see if an `ResourceNotFoundException` is
+      thrown, which indicates that the workflow cannot be found. If the
+      exception is thrown, display the message and end the program.
+
+5. **List Matching Workflows**:
+    - Description: Lists all matching workflows created within the account by
+      calling the `listMatchingWorkflows` method.
+    - Exception Handling: Check to see if an `CompletionException` is thrown. If
+      so, display the message and end the program.
+
+6. **Get Schema Mapping**:
+    - Description: Returns the `SchemaMapping` of a given name by calling the
+      `getSchemaMapping` method.
+    - Exception Handling: Check to see if a `ResourceNotFoundException` is
+      thrown. If so, display the message and end the program.
+
+7. **Tag Resource**:
+    - Description: Adds tags associated with an AWS Entity Resolution resource
+      by calling the`tagResource` method.
+    - Exception Handling: Check to see if a `ResourceNotFoundException` is
+      thrown. If so, display the message and end the program
+8. **Delete Matching Workflow**:
```

Comment:
The code that runs says "8. View the results of the AWS Entity Resolution Workflow.", which doesn't match this step in the Spec.

Reply:
fixed